tests/stat/test_fstatat.c

```diff
@@ -134,6 +134,14 @@ void test() {
   assert(s.st_blocks == 1);
 #endif
 
+  struct timespec origTimes[2];
+  origTimes[0].tv_sec = (time_t)s.st_atime;
+  origTimes[0].tv_nsec = origTimes[0].tv_sec * 1000;
+  origTimes[1].tv_sec = (time_t)s.st_mtime;
+  origTimes[1].tv_nsec = origTimes[1].tv_sec * 1000;
+  while ((err = futimens(fd, origTimes)) < 0 && errno == EINTR);
```

Comment:
Do you need this loop?   I don't think this should ever fail with EINTR... I'm not even sure we have any EINTR in emscripten at all.

Reply:
I wonder if we should put this in a new test file called `test_futimens.c`.. would that be easy?

Reply:
Added test file test_futimens.c